devfiles/lombok/devfile.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: 1.0.0
+metadata:
+  generateName: javalombok-
+projects:
+  - name: lombok-demo
+    source:
+      location: 'https://github.com/che-samples/lombok-demo.git'
```

Comment:
This git repo is empty

Reply:
[demo pr](https://github.com/che-samples/lombok-demo/pull/1)